data/extension/init/templates/tinygo/envoy.access_loggers/default/go.mod

```diff
@@ -0,0 +1,8 @@
+module my.filters.http
```

Comment:
[nit] If `(Tiny)Go` is fine with module names like `my.filters.http`, can we reuse here value collected by the `getenvoy extension init` ?

E.g., if a user chose
```shell
What kind of extension would you like to create?
✔ Category HTTP Filter
✔ Language TinyGo
✔ Output directory http_example
✔ Extension name mycompany.filters.http.example
```
, can we reuse `mycompany.filters.http.example` in the generated `go.mod` ?

Reply:
Thanks. done: https://github.com/tetratelabs/getenvoy/pull/104/commits/45a02f3d8be384ae359161c39478e4d2c7f6de15